### ISSUE TITLE
Improve Satellite view perfomance

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#200]
## Description
Currently the FE requests both the satelliite and dop40 images for bavaria. To reduce overhead the low res satellite images should only be shown for regions without dop40 images, otherwise only the dop40 should be displayed

## Acceptance criteria
* Display dop40 images for bavaria
* Display satellite images for regions without dop4 images (rest of germany)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
